NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs

```diff
@@ -221,5 +211,27 @@ public override void OnBackPressed()
             }
             base.OnBackPressed();
         }
+
+        private void SetSwitchButtonTextColor ()
```

Comment:
`_switchCustom.SetSwitchTextAppearance(this, _switchCustom.Checked ? Resource.Style.SwitchTextAppearanceOn : Resource.Style.SwitchTextAppearanceOff);`

I guess can be shortened like this to avoid if/else construction